### PR TITLE
Refresh most cURL examples and extensive edits

### PR DESCRIPTION
**Pull request by brewdium**

### CODE REVIEW

File: guides/items-and-slots.md [hunk 1]
```diff
@@ -1,39 +1,36 @@
...
 
-Because creating an Item begins with an Item template, we can begin with listing all available Item templates:
+Items are created from templates, so we begin by listing all available Item templates:
 
 ```bash
-    curl --request GET \
-  --url https://sandbox.meeco.me/vault/item_templates \
-  -H 'authorization: Bearer VAULT_ACCESS_TOKEN_FROM_CLI_GENERATED_USER' \
-  -H 'Meeco-Subscription-Key: DEV_PORTAL_SUBSCRIPTION_KEY'
+curl --request GET 'https://vault-stage.meeco.me/item_templates' \
```
Comment (DoubleDroz) [inline guides/items-and-slots.md]: Keep URL as `sandbox.meeco.me/vault`

File: guides/items-and-slots.md [hunk 2]
```diff
@@ -221,177 +233,306 @@ Here is a response with a created Item:
...
-curl -v -X POST "https://sandbox.meeco.me/vault/item_templates"
--H "Content-Type: application/json"
--H "Cache-Control: no-cache"
--H "Meeco-Subscription-Key: DEV_PORTAL_SUBSCRIPTION_KEY"
--H "authorization: Bearer VAULT_ACCESS_TOKEN_FROM_CLI_GENERATED_USER"
---data-ascii "{
-  ^"name^": ^"example_custom_template^",
-  ^"label^": ^"Example Custom Template^",
-  ^"slots_attributes^": [
+    curl --request POST "https://vault-stage.meeco.me/item_templates" \
```
Comment (DoubleDroz) [inline guides/items-and-slots.md]: Keep url as `sandbox.meeco.me`

File: guides/items-and-slots.md [hunk 3]
```diff
@@ -221,177 +233,306 @@ Here is a response with a created Item:
...
-    ^"item^": {
-      ^"label^": ^"An example item with custom template^",
-      ^"classification_nodes_attributes^": [
-      ],
-      ^"slots_attributes^": [
-      ]
-    },
-    ^"template_name^": ^"example_custom_template^"
-  }"
+  curl --request POST "https://vault-stage.meeco.me/items" \
```
Comment (DoubleDroz) [inline guides/items-and-slots.md]: keep url as `sandbox.meeco.me`

File: guides/items-and-slots.md [hunk 4]
```diff
@@ -221,177 +233,306 @@ Here is a response with a created Item:
...
+-   `password`
+
+As the Vault cannot inspect the data, it is just a suggestion to the user. Type `key_value` is the default.
+
+Slots are created either by cloning an Item Template, or via the `slots_attributes` property when creating an Item. Since they are keyed by `name`, either `label` or `name` must be non-empty on creation.
+
+Slots are updated by calling `PUT /vault/items` with the new data in `slots_attributes`:
+
+```bash
+curl --request PUT "https://vault-stage.meeco.me/items/bef961af-aa1f-4f1c-ac95-cdb41b3682db" \
```
Comment (DoubleDroz) [inline guides/items-and-slots.md]: keep url as `sandbox.meeco.me`

File: guides/items-and-slots.md [hunk 5]
```diff
@@ -221,177 +233,306 @@ Here is a response with a created Item:
...
+### Slots
+
+Slots represent a key-value pair, where the value is always encrypted. The Meeco Vault will check and reject any `encrypted_value` data that doesn't match the cryppo serialization format.
+
+Slots are always owned by an Item (or an Item Template, but these Slots are never read), and news Slots are created for each new Item.
+
+Their most important properties are:
+
+| Property          | Description                 |
+|----------------- |--------------------------- |
```
Comment (DoubleDroz) [inline guides/items-and-slots.md:462]: missed a space :)

File: guides/items-and-slots.md (hunk 3 above)
Comment (javereec) [reply guides/items-and-slots.md]: We shouldn't use vault-stage.meeco.me

**Pull request merged**